### PR TITLE
boards: define default radio for wsn430 boards

### PR DESCRIPTION
**Pull request by miri64**
### Contribution description
While testing #8061 I noticed, that the default radios for the wsn430 are not configured. Sadly it doesn't work right now but it doesn't work, but I don't know why. At least with my limited possibilities (debugging via IoT-Lab + using `printf()`) I was able to see, that

* the wsn430-v1_3b hangs after calling `cc110x_cs()` in `_reset()` of `drivers/cc110x/cc110x.c` (called from `cc110x_setup()` in `auto_init`) and
* the wsn430-v1_4 hangs after `gpio_set(dev->params.pin_reset);` in the `netdev::init()` method of the cc2420 driver.

Anyone has an idea? Maybe some from the INRIA people @kYc0o @aabadie or one of the more hardware-savvy crowy @PeterKietzmann @gebart?

### Issues/PRs references
Depends on #8437.

### CODE REVIEW

File: boards/wsn430-v1_3b/include/board.h [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#endif
+
+/**
+ * @brief   Definition of the interface to the CC1101 radio
+ */
+#define CC110X_PARAMS_BOARD     {   .spi  = SPI_DEV(0),     \
+                                    .cs   = GPIO_PIN(P4, 1), \
+                                    .gdo0 = GPIO_PIN(P1, 3), \
+                                    .gdo1 = GPIO_PIN(P5, 2), \
+                                    .gdo2 = GPIO_PIN(P1, 4) }
```
Comment (miri64) [inline boards/wsn430-v1_3b/include/board.h]: I've taken those from the [WSN430 1.3b schematics](https://github.com/iot-lab/iot-lab/wiki/Hardware_Wsn430-node#schematics-and-datasheets).

File: boards/wsn430-v1_4/include/board.h [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+ */
+#define CC2420_PARAMS_BOARD     {   .spi        = SPI_DEV(0),       \
+                                    .spi_clk    = SPI_CLK_5MHZ,     \
+                                    .pin_cs     = GPIO_PIN(P4, 2),  \
+                                    .pin_fifo   = GPIO_PIN(P1, 3),  \
+                                    .pin_fifop  = GPIO_PIN(P1, 4),  \
+                                    .pin_cca    = GPIO_PIN(P1, 6),  \
+                                    .pin_sfd    = GPIO_PIN(P1, 5),  \
+                                    .pin_vrefen = GPIO_PIN(P3, 0),  \
+                                    .pin_reset  = GPIO_PIN(P1, 7)   }
```
Comment (miri64) [inline boards/wsn430-v1_4/include/board.h]: I've taken those from the [WSN430 1.4 schematics](https://github.com/iot-lab/iot-lab/wiki/Hardware_Wsn430-node#schematics-and-datasheets).

**Comment by miri64** [comment]
Rebased to current #8437 and adapted also for the new parameter definition scheme.

**Comment by PeterKietzmann** [comment]
This PR needs rebase after #9124 was merged and #8437 was closed. As long as this PR is just the matter of configuration I'd give my ACK with looking in the datasheet. For the actual bug @maribu might have a clue:

>Sadly those timing issues aren't the only problem. It seems the SPI transfer to the MSB IoT is not working completely reliable. I hope to figure that out as well.

**Comment by miri64** [comment]
> @miri64: We did a lot of duplicated work. Sorry for that, I wasn't aware of your PR :-(

nP, shit happens ;-)

File: boards/wsn430-v1_3b/include/board.h [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Define the interface to the CC1101 radio
+ * @{
+ */
+#define CC110X_PARAMS_CS            (GPIO_PIN(P4, 1))
```
Comment (PeterKietzmann) [inline boards/wsn430-v1_3b/include/board.h]: Please check this pin. If i read the schematics correctly it is P4_2

Comment (miri64) [reply boards/wsn430-v1_3b/include/board.h]: You're right!

**Comment by miri64** [comment]
> Otherwise I suggest to squash to two commits. One for shift to "common" and the other to introduce configurations

If you insist on spliting the common and configuration, I would prefer a third for the examples modification ;-)

**Comment by miri64** [comment]
> Did you verify the pin?

In https://github.com/RIOT-OS/RIOT/pull/8438#discussion_r191539847

> You're right!

Confirmed and re-confirmed ;-)

**Comment by kYc0o** [comment]
AFAIK, we don't have a mean to initialise the second UART as an SPI device on wsn430 nodes. In #5599 I opened an issue which states that we don't have configuration for SPI yet, but it's not straightforward to implement since it's a shared peripheral with UART. In #5606 there's an effort to configure it but AFAIR it doesn't work properly.

**Comment by PeterKietzmann** [comment]
@kYc0o do you propose an alternative? The shared bus is a problem - I agree. But however, in a "deployment" I'd rather abstain from STDIO over UART that a configured radio. That's why I'd go with this PR. And we can not change the config anyway...

**Comment by PeterKietzmann** [comment]
@miri64 
```
	boards/wsn430-v1_3b/Makefile.dep:4: new blank line at EOF.
	boards/wsn430-v1_4/Makefile.dep:4: new blank line at EOF.
	ERROR: This change introduces new whitespace errors
```

**Comment by miri64** [comment]
> @kYc0o do you propose an alternative? The shared bus is a problem - I agree. But however, in a "deployment" I'd rather abstain from STDIO over UART that a configured radio. That's why I'd go with this PR. And we can not change the config anyway...

Well that would explain my initial problems I wrote down in OP, so I rather would agree with @kYc0o here to not merge it now. @PeterKietzmann Did you test it on IoT-LAB and confirmed that it work?

**Comment by miri64** [comment]
(and if yes, can you tell me how?)

**Comment by PeterKietzmann** [comment]
@miri64 as described above I only did a data sheet comparison for the pin config. The above referenced issue isn't new but does not conflict with adding proper pin configurations IMO. However, after thinking again about it I propose we take out the applications Makefile and the boards Makefile.dep changes to proceed with this PR. The rest we leave up for @kYc0o (or @thomaseichinger) in  #5606

**Comment by miri64** [comment]
Ok. Done.

**Pull request merged**